### ISSUE TITLE
Follower count -1

### ISSUE THREAD

**Issue by ThatOneCalculator** [syuilo/Misskey#8552]
## 💡 Summary

I was blocked by this user and now their follower count shows -1

![image](https://user-images.githubusercontent.com/44733677/165419289-36a3971d-2946-433f-b8bc-8e81e38083bf.png)

## 📌 Environment

Misskey version: 12.110.1
Your OS: Linux
Your browser: Firefox

**Comment by Johann150** [comment]
This is also a duplicate of #8516. The followers count is not decremented if there are no followers. So I assume the followers count did not get incremented properly before but did get decremented properly now. https://github.com/misskey-dev/misskey/blob/5ad42d1d855872d629fb361a9eea47260c4401e9/packages/backend/src/services/blocking/create.ts#L88-L96
